variants/ESPDuino/pins_arduino.h

```diff
@@ -33,7 +33,7 @@
 static const uint8_t SDA = PIN_WIRE_SDA;
 static const uint8_t SCL = PIN_WIRE_SCL;
 
-static const uint8_t LED_BUILTIN = 16;
-static const uint8_t BUILTIN_LED = 16;
+#define LED_BUILTIN 16
```

Comment:
Could a compiler #define such as "-DBUILTIN_LED=LED_BUILTIN" and remove the need for all these repeated 2nd #define BUILTIN_LED X?

Reply:
BUILTIN_LED is not used in core, but seems to be historically used in esp8266 code outside.
What about moving this define into ```variants/generic/common.h``` (hidden, wrong name less exposed, less loaded gcc cmdline) ?

Reply:
If it's not standard, how about putting it somewhere safe and deprecating its use?

Reply:
Good idea